packages/dapp/pages/bonds.tsx

```diff
@@ -1,5 +1,7 @@
 import { FC } from "react";
-import BondsContainer from "../components/bonds/App";
+
+import dynamic from "next/dynamic";
+const BondsContainer = dynamic(() => import("../components/bonds/App"), { ssr: false }); //@note Fix: (Hydration Error)
```

Comment:
Can you explain to me why this solves the problem? For my education.

Reply:
well it's slightly complicated but in general, Nextjs renders the page on the server and hydrates on the client
if those versions don't match (due to dynamic rendering such as based on the isConnected/walletAddress)
it throws the hydration error
here's a nice article explaining this in more detail
https://jonmeyers.io/blog/fix-client-server-hydration-error-in-next-js
@pavlovcik